lifecycle-operator/chart/templates/keptnappversion-crd.yaml

```diff
@@ -4,7 +4,10 @@ metadata:
   name: keptnappversions.lifecycle.keptn.sh
   annotations:
     controller-gen.kubebuilder.io/version: v0.13.0
-    cert-manager.io/inject-ca-from: '{{ .Release.Namespace }}/keptn-certs'
+    {{- $caAnnotations := .Values.global.caInjectionAnnotations}}
```

Comment:
since the annotations: is anyhow added there is no need for this complexity, you can just include the values here